backend/exceptions/db_exceptions.py

```diff
@@ -0,0 +1,4 @@
+class RecipeAlreadyExist(ValueError):
+    """Raise when a certain recipe name already exist in the database"""
+    def __init__(self) -> None:
+        super().__init__('Recipe already exists in the database')
```

Comment:
Maybe we should also include name of recipe here

Reply:
Done